### PR TITLE
Added information about Prometheus configuration

### PR DESCRIPTION
**Pull request by GoldenCorgo**
Recommendation on reducing metrics ingestion and warning about KSM emission overlapping.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ helm install my-prometheus --repo https://prometheus-community.github.io/helm-ch
   -f https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/prometheus/extraScrapeConfigs.yaml
 ```
 
+This Prometheus installation is based on Prom community helm chart, and by default your Prometheus will be scraping a lots of unneccessary metrics and taking quite a lot of space. For production purposes you could refer to the Kubecost [user metrics list](https://guide.kubecost.com/hc/en-us/articles/4425134686743-User-Metrics) to filter with 'keep', for reference take a look at scraping config at the ConfigMap section of Kubecost installation [chart](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/kubecost.yaml).
+
+If you are going to conect existing Prometheus instance which is already consuming KSM metrics, please consider visiting this page about [KSM metrics emission](https://guide.kubecost.com/hc/en-us/articles/4408095797911), because Opencost currently implements the same architecture and you maight get overlapping metrics.
```
Comment (AjayTripathy) [inline docs/install.md:37]: Nit: "maight" is spelled "might"

File: docs/install.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ helm install my-prometheus --repo https://prometheus-community.github.io/helm-ch
   -f https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/prometheus/extraScrapeConfigs.yaml
 ```
 
+This Prometheus installation is based on Prom community helm chart, and by default your Prometheus will be scraping a lots of unneccessary metrics and taking quite a lot of space. For production purposes you could refer to the Kubecost [user metrics list](https://guide.kubecost.com/hc/en-us/articles/4425134686743-User-Metrics) to filter with 'keep', for reference take a look at scraping config at the ConfigMap section of Kubecost installation [chart](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/kubecost.yaml).
```
Comment (AjayTripathy) [inline docs/install.md:35]: Here's a bit of a rewrite for clarity.
```
This Prometheus installation is based on Prom community helm chart, and by default your Prometheus might scrape unnecessary metrics. For production, you can refer to the Kubecost [user metrics list](https://guide.kubecost.com/hc/en-us/articles/4425134686743-User-Metrics) to filter with 'keep', for reference take a look at scrape config in the Kubecost installation [chart](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.98/cost-analyzer/charts/prometheus/values.yaml#L1208).

File: docs/install.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ helm install my-prometheus --repo https://prometheus-community.github.io/helm-ch
 kubectl apply --namespace opencost -f https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/opencost.yaml
 ```
 
-## Prerequisites
+## Prerequisites: Prometheus
 
-Install Prometheus using the following command:
+Opencost is relying on metrics scraped by Prometheus. For express installation of Prometheus use the following command:
```
Comment (AjayTripathy) [inline docs/install.md:25]: s/is relying/relies

**Review by AjayTripathy — CHANGES_REQUESTED** [review]
Couple of grammar edits but thank you for this idea!

File: docs/install.md (hunk 1 above)
Comment (GoldenCorgo) [reply docs/install.md:37]: Definitely should resist contributing too late at night.

**Review by AjayTripathy — APPROVED** [review]

**Pull request merged**